apps/frontend/src/components/Auth/apiClient.tsx

```diff
@@ -67,7 +67,7 @@ export class ApiClient {
   }
 
   public async updateTimesheet(req): Promise<unknown> {
-    return this.axiosInstance.post('/auth/timesheet', req)
+    return this.axiosInstance.post("/auth/timesheet", req);
```

Comment:
Does ' vs " make a difference?

Reply:
whoops no I think that's my prettier settings